Makefile

```diff
@@ -1,25 +1,27 @@
-.PHONY: all install build libcswsscommon
+.PHONY: all install build libcswsscommon clean
+
+GO := /usr/local/go/bin/go
+export GOROOT=/usr/local/go
+export GOPATH=$(HOME)/go
+export GOBIN=$(GOPATH)/bin
 
 all: install build
 
 install: build
-	install -D $(GOPATH)/bin/go-server-server $(DESTDIR)/usr/sbin/go-server-server
+	/usr/bin/install -D $(GOPATH)/bin/go-server-server debian/sonic-rest-api/sbin/go-server-server
```

Comment:
This and other changes seem to generate the go-server-server at /sbin/ vs /usr/sbin/ that it was originally. The supervisor files would need to be modified to look for go-server-server at sbin in order to run the rest-api server.